### PR TITLE
Updated Labs to use latest rhte lab

### PR DESCRIPTION
**Pull request by mvazquezc**

### CODE REVIEW

File: labs/1.md [hunk 1]
```diff
@@ -5,24 +5,35 @@ This hands on lab is a simple deployment of [Kubefed Operator](https://operatorh
 clusters. This lab will use the KubeFed operator above as well as the binary `kubefedctl`.
 We will deploy cluster scoped KubeFed and deploy applications to multiple clusters using KubeFed Primitives.
 
+<a id="markdown-accesing-the-clientvm" name="accesing-the-clientvm"></a>
+# Accesing the ClientVM
+All the work will be done using the ClientVM that you get from GuidGrabber for Cluster1 Environment. Let's go ahead an login into the ClientVM. Remember that login details are presented by GuidGrabber.
```
Comment (cooktheryan) [inline labs/1.md]: small NIT

All of the work will be completed using the ClientVM

File: labs/1.md [hunk 2]
```diff
@@ -5,24 +5,35 @@ This hands on lab is a simple deployment of [Kubefed Operator](https://operatorh
...
+
+e.g: ssh student@clientvm.d5fe.sandbox253.opentlc.com
+~~~
+
 <a id="markdown-pre-requisites" name="pre-requisites"></a>
 # Prerequisites
-Three OpenShift clusters have already been deployed for use in this lab. During
-this lab we will utilize the `kubefedctl` binary to join the three clusters
+At this point you should be logged in ClientVM and you should have all three environment information from GuidGrabber.
+
```
Comment (cooktheryan) [inline labs/1.md]: in to the ClientVM and you should have the information for all three environments defined by the GuidGrabber.

File: labs/1.md [hunk 3]
```diff
@@ -32,14 +43,31 @@ kubefedctl version
...
+~~~sh
+cfssljson -help
+
+Usage of cfssljson:
+<OUTPUT OMITTED>
+~~~
+
 
-Clone the demo code to your local machine:
+Verify that example code is present in the `clientVM`:
```
Comment (cooktheryan) [inline labs/1.md]: Verify that the lab content is available on the ClientVM

File: labs/2.md [hunk 4]
```diff
@@ -1,35 +1,28 @@
 <a id="markdown-configure-client-context-for-cluster-admin-access" name="configure-client-context-for-cluster-admin-access"></a>
 ## Configure OpenShift client context for cluster admin access
 
-The installer has created a `kubeconfig` file for each cluster, we are going to merge them in the same file so we can use that file later with `kubefedctl` tool.
+You should have three environments from GuidGrabber, each environment has a different **OpenShift API for command line 'oc' client** config, make sure you
+have three different API Endpoints ready for use. API Endpoints look like: `https://api.cluster-d5fe.d5fe.sandbox253.opentlc.com:6443`
 
-First, we will rename the `admin` context and then we will rename the admin user so when we merge the two `kubeconfig` files both admin users are present.
+Now, we are going to login into each cluster an rename the context. Make sure you use the correct cluster information for each login step. First login have
```
Comment (cooktheryan) [inline labs/2.md]: each cluster and rename

Comment (cooktheryan) [reply labs/2.md]: For example the first cluster login will be **A000....

File: labs/6.md [hunk 5]
```diff
@@ -15,13 +15,106 @@ Below the architecture definition for our Pacman Application.
...
+    ```
+
 ## Deploying the Pacman Application
 
 Now that the Mongo cluster has been configured, it is time to deploy the *pacman* application.
 There are many different types of federated objects but they are somewhat similar to those
 non-federated objects. For more information about federated objects see the following  [examples](https://github.com/kubernetes-sigs/kubefed/tree/master/example/sample1) and
 the [user guide](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md).
 
+First, change directory to the Pacman demo folder.
```
Comment (cooktheryan) [inline labs/6.md]: change to the directory of the Pacman demo folder

File: labs/7.md [hunk 6]
```diff
@@ -55,13 +60,27 @@ To populate the application back to all clusters, modify the placement to includ
 oc --context=cluster1 -n pacman patch federateddeployment pacman --type=merge -p '{"spec":{"placement":{"clusters": [{"name":"cluster1"}, {"name":"cluster2"}, {"name":"cluster3"}]}}}'
 ~~~
 
-Can you spot the difference between scaling a deployment to 0 in a cluster using `overrides` and removing a deployment from a cluster using `placement`?
+Verify deployments are created again in all three clusters.
```
Comment (cooktheryan) [inline labs/7.md]: Verify the deployments are created again in all three

File: labs/2.md [hunk 7]
```diff
@@ -4,8 +4,10 @@
 You should have three environments from GuidGrabber, each environment has a different **OpenShift API for command line 'oc' client** config, make sure you
 have three different API Endpoints ready for use. API Endpoints look like: `https://api.cluster-d5fe.d5fe.sandbox253.opentlc.com:6443`
 
-Now, we are going to login into each cluster an rename the context. Make sure you use the correct cluster information for each login step. First login have
-to be **A0004-A** environment, second login **A0004-B** and third login **A0004-C**.
+Now, we are going to login into each cluster and rename the context. Make sure you use the correct cluster information for each login step. For example the first login will
+be **A0004-A** environment, second login **A0004-B** and third login **A0004-C**.
+
+> **NOTE:** Below commands refer to GUID and DOMAIN. GUID refers to the environment uniq id assigned by GuidGrabber and DOMAIn refers to the environment dns domain assigned by GuidGrabber. e.g: GUID - `d5fe` DOMAIN - `sandbox253.opentlc.com`
```
Comment (sabre1041) [inline labs/2.md]: Lowercase `n` in DOMAIN

**Comment by mvazquezc** [comment]
Squashed commits. This should be ready for final review @cooktheryan @sabre1041

**Comment by mvazquezc** [comment]
Added WIP again since we will need to modify some files after merging  https://github.com/openshift/federation-dev/pull/61

File: labs/1.md [hunk 8]
```diff
@@ -1,46 +1,72 @@
...
 clusters. This lab will use the KubeFed operator above as well as the binary `kubefedctl`.
 We will deploy cluster scoped KubeFed and deploy applications to multiple clusters using KubeFed Primitives.
 
-<a id="markdown-pre-requisites" name="pre-requisites"></a>
-# Prerequisites
-Three OpenShift clusters have already been deployed for use in this lab. During
-this lab we will utilize the `kubefedctl` binary to join the three clusters
-as well as enable the required types and then use the `oc` command to deploy objects used for the lab.
+<a id="markdown-accesing-the-clientvm" name="accesing-the-clientvm"></a>
+# Accesing the ClientVM
```
Comment (cooktheryan) [inline labs/1.md]: Accessing

File: labs/11.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+Thanks for attending this hands on lab. By now you have completed the following activities.
+
+* Explored the environment
+* Configured oc tool to access multiple clusters with custom context names
+* Deployed KubeFed
+* Deployed a Federated WebServer Application
+* Deployed a Federated MongoDB ReplicaSet
+* Deployed an HAProxy to provide global ingress for Pacman
+* Deployed a Federated Pacman
+* Learnt about application portability using KubeFed primitives
```
Comment (cooktheryan) [inline labs/11.md]: Learned 
* i think this is just geographic based preference

File: labs/3.md [hunk 10]
```diff
@@ -32,6 +32,7 @@ EOF
 The KubeFed Controller should be deployed now:
 
 ~~~sh
+wait-for-deployment cluster1 kube-federation-system kubefed-controller-manager
```
Comment (cooktheryan) [inline labs/3.md]: is this in the wrong spot?

File: labs/intro.md [hunk 11]
```diff
@@ -1,111 +1,61 @@
...
-image:vmconsole.png[200,200]
-
-Congratulations, you got to your *graphical console*.
-
-
-## Using the text console to access the remote shell
-
-On your laptop (i.e. not in the browser window with the remote deskop), open the `Terminal` application.
-Then, enter the following command, replacing `GUID` with your actual GUID:
+### CHANGEME URL for RHTE will be generic_rhte
```
Comment (cooktheryan) [inline labs/intro.md]: Do you need to change this now before PR?

File: labs/1.md (hunk 8 above)
Comment (mvazquezc) [reply labs/1.md]: fixed.

File: labs/3.md (hunk 10 above)
Comment (mvazquezc) [reply labs/3.md]: No, it's okay there. We should wait until the deployment is ready in order to see the pods running.

File: labs/intro.md (hunk 11 above)
Comment (mvazquezc) [reply labs/intro.md]: Nope, we need to change it before moving the lab to GA.

**Pull request merged**